WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2111,3 +2167,63 @@ CircularProgressBarTextColor        = @TxtC@;
+CheckElmStyles                      = display:inline-block; vertical-align:middle; margin:0; padding:0; position:relative; @FixBoxModel@; width:@CheckElmSize@; height:@CheckElmSize@; @ActiveCursor@; @appearance@:none; overflow:hidden; outline:none; @TransparentBg@; border:1px solid @CheckElmBorderColor@;
+CheckElmTickIconStyles              = content:''; position:absolute; display:block; @FixBoxModel@; opacity:0;
+
+# Checkbox styles
+CheckBox                    = @CheckElmStyles@; @MediumRoundCorners@;
+CustomCheckBoxTickIcon      = @CheckElmTickIconStyles@; @TransparentBg@; top:8%; left:30%; width:40%; height:70%; border:2px solid @CheckElmTickColor@; border-top:none; border-left:none; @transform@:rotate(40deg);
+CheckBoxChecked             = background-color:@CheckElmActiveBgColor@; border-color: @CheckElmActiveBorderColor@; box-shadow:none;
+CheckBoxCheckedTickIcon     = opacity:1;
+CheckBoxActive              = border-color:@CheckElmActiveBorderColor@; box-shadow:none;
+CheckBoxDisabled            = opacity:0.4;
+CheckBoxLabel               = display:inline-block; vertical-align:middle; margin-left:8px;
+
+# Radio button styles
+RadioButton                 = @CheckElmStyles@; @CompleteRoundCorners@;
+RadioButtonTickIcon         = @CheckElmTickIconStyles@; background:@CheckElmActiveBgColor@; top:21%; left:21%; width:60%; height:60%; @CompleteRoundCorners@;
+RadioButtonChecked          = @CheckBoxChecked@; @TransparentBg@;
+RadioButtonCheckedTickIcon  = @CheckBoxCheckedTickIcon@;
+RadioButtonActive           = @CheckBoxActive@;
+RadioButtonDisabled         = @CheckBoxDisabled@;
+RadioButtonLabel            = @CheckBoxLabel@
+
+######################################
+#   PREFERENCES -APP SPECIFIC STUFF
+######################################
+
+PrefViewWrapperPadding      = @SkinViewContentPadding@
+PrefViewFontSize            = @FontSize-slightly-big@
+PrefViewGutterSpace         = @SkinViewGutterSpace@
+PrefViewInnerSpacing        = @SkinViewInnerSpacing@
+
+PrefView                    = @FixBoxModel@; @FullWidth@; padding:@PrefViewWrapperPadding@; @PrefViewFontSize@;
+PrefSectionHeaderWrapper    = display:flex; justify-content:space-between; align-items:center;
+PrefSectionHeader           = @FontSize-slightly-bigger@; @FixBoxModel@; @FullWidth@; padding:@PrefViewGutterSpace@ 0;
+PrefSectionHeaderActions    = @PrefSectionHeaderWrapper@
+PrefSectionContentTable     = margin:0 0 @PrefViewWrapperPadding@ 0;
+PrefSectionContent          = padding:0; margin:0;
+PrefSectionInternalSpacing  = 4px
+PrefSectionRowCell          = padding:@PrefSectionInternalSpacing@ 0;
+PrefSectionRowLabelWrapper  = width:30%; min-width:200px; @PrefSectionRowCell@;
+PrefSectionRowValueWrapper  = width:70%; @PrefSectionRowCell@;
+PrefSectionRowLabel         = @Text@; width:150px; white-space:nowrap; vertical-align:middle; line-height:20px;
+PrefRowCheckWidget          = padding:2px 0;
+PrefRowSelectWidget         = min-width:280px; width:auto; max-width:100%;
```

Comment:
Well not actually. Just added for semantic reason. As we have use case as follows:

1. Dropdown should be of atleast 280px wide
2. If it's content is more than 280px wide, it's width should be auto
3. However width should not exceeds 100% of containers width

I can remove `width:auto` here. Let me know.

Thanks